README.rst

```diff
@@ -100,6 +100,7 @@ Usage
                             number of answers to return
       -C, --clear-cache     clear the cache
       -v, --version         displays the current version of howdoi
+      -e [ENGINE], --search-engine [ENGINE]  change search engine for this query only
```

Comment:
Here I would list the possible engines (google, bing)

Reply:
Also `--engine` is fine for the long name

Reply:
https://github.com/gleitz/howdoi/pull/235/commits/c1df93fc892761377698738d6089b72b3949b076